icx/wallet/wallet.py

```diff
@@ -220,4 +221,35 @@ def get_address(self):
         """
         return self.address
 
+    @staticmethod
+    def get_block_by_height(height, uri="https://testwallet.icon.foundation/api/"):
+        """ get block information by height
+
+        :param height:
+        :param uri type(str)
+        :return:
+        """
+        block = get_block_by_height(height, uri)
+        return block
+
+    @staticmethod
```

Comment:
Use @classmethod, not @staticmethod. See [this](https://github.com/shoark7/Effective-Python/blob/master/files/BetterWay24_classmethod.md) document.

Reply:
Fixed it.